### PR TITLE
Release 5.6.2

### PR DESCRIPTION
**Pull request by velzevur**

### CODE REVIEW

**Review by hanssv — DISMISSED** [review]

**Review by gorbak25 — DISMISSED** [review]

File: apps/aecore/test/aec_mining_tests.erl [hunk 1]
```diff
@@ -40,10 +40,10 @@ mine_block_test_() ->
                  %% let_it_crash = generate_valid_test_data(TopBlock, 100000000000000),
                  Nonce = case aec_hard_forks:protocol_effective_at_height(Height + 1) of
                              ?ROMA_PROTOCOL_VSN    -> 1157794539819639234;
-                             ?MINERVA_PROTOCOL_VSN -> 17834638850274440624;
```
Comment (radrow) [inline apps/aecore/test/aec_mining_tests.erl:43]: Why these changes?

Comment (velzevur) [reply apps/aecore/test/aec_mining_tests.erl:43]: This test checks we can find a solution with a known-to-be-working nonce. Since we change the block header info, this changes the hash of the block candidate we base the puzzle on. Hence "find a solution" test shall be adapted accordingly.

**Review by radrow — APPROVED** [review]

**Review by uwiger — APPROVED** [review]

**Review by dincho — APPROVED** [review]
